include/bout/paralleltransform.hxx

```diff
@@ -6,6 +6,7 @@
 #ifndef __PARALLELTRANSFORM_H__
 #define __PARALLELTRANSFORM_H__
 
+#include <datafile.hxx>
```

Comment:
As we only need a reference to `Datafile` in this header, you could just forward declare it:

```cpp
class Datafile;
```

This avoids needing to `#include` the header in this header, hopefully speeding up compilation times. It will need to go in `shiftedmetric.cxx` so `ShiftedMetric::outputVars` can use `Datafile::add`